### DISCUSSION TITLE
Support the project!

### DISCUSSION THREAD

**Discussion by nealmick** [nealmick/bb#9] [Announcements]
![Screen Shot 2023-03-09 at 1 32 36 PM](https://user-images.githubusercontent.com/33704976/224121930-2294f949-dec9-4dba-860a-3685898d53d9.png)

Currently this project is completely free.  Subscribing helps to support ongoing development and pays to keep the live server running!  I really want to upgrade the live server, so it can handle at-least some light training.  I would also like to build out multiple model slots, so each user could have up to 3 or more models saved at a time.  There are also a lot more model settings i want to build soon, an early stopping toggle, with restore weights option, and other things like kernel regularizers.

 I greatly appreciate any support and hope to build many more features soon!
https://nbadata.cloud/shop/

**Comment by DemonButler** [comment]
Will be supporting! Keep up the good work!

**Reply by nealmick** [reply]
Hey Thanks a bunch!! let me know if you have any ideas or features you would like to see!
